package test.task.vacancyparser.exception;

public class UrlConnectionException extends RuntimeException {
    public UrlConnectionException(String message, Throwable cause) {
        super(message, cause);
    }
}
